docs/en/guides/webhooks.md

```diff
@@ -0,0 +1,198 @@
+
+### Create a directory structure on disk when an entity is created
+
+One workflow that we have seen repeated many times over is the need to create a directory structure on disk when a new entity is created in Shotgun. Why not go ahead and make sure that new Shot is ready for artist work automatically once it is created in Shotgun?
+
+### Automation of status management
+
+When your animation team is finished with their work, why not go ahead and change the status of downstream Tasks in the same Shot to indicate that more work is ready to begin? This way, you can automatically notify the assigned artists in different disciplines that everything is ready to go!
+
+Another great example of how to automate status management would be to trigger a status change on a Task when a new Note is created. This is a good way to indicate to the artist and production teams that a supervisor has requested changes or fixes to the current work after a review session.
+
+## When should webhooks be used instead of the Shotgun event daemon?
+
+Webhooks and the [Shotgun event daemon](https://github.com/shotgunsoftware/shotgunEvents/wiki) offer similar features, but with a few key differences. The event daemon requires that you run, monitor, and maintain your own service. All of your code must be written in Python, and it allows you to initiate your own connections to Shotgun. Webhooks, in contrast, answer connections and can be written in any programming language. They can be hosted in a serverless environment, such as [AWS Lambda](https://aws.amazon.com/lambda/), or can trigger any of the automation platforms available online, such as [Zapier](https://zapier.com) and [IFTTT](https://ifttt.com). If your use case works with webhooks, it should be the preferred solution.
+
+## Creating a webhook
+
+To get started creating a webhook, go to a Webhooks page, then navigate to the button above the webhooks list.
+
+![Create Webhook Button](./images/webhooks/create_webhook_button.png)
+
+Next up is to fill out the information required to create the new webhook.
+
+![Create Webhook Dialog](./images/webhooks/create_webhook_dialog.png)
+
+### Secret token
+
+Assigning a secret token to a webhook is optional. If provided, any request sent to the webhook URL will be signed using that token. The token value is sent with the request as a header named `X-SG-SIGNATURE`. The signature is calculated using HMAC with SHA1 and the message signed is the JSON body of the request.
+
+#### Header format
+
+`<algorithm>=<signature>`
+
+#### Why use a secret token?
+
+While not strictly required, providing a secret token causes the payload sent to the webhook URL to be signed. This allows the consumer service to verify that the data originated from the source it expects, and that the payload has not been altered in any way during transit.
+
+#### Signature verification
+
+An example of how to verify the signature of the payload is provided below using Python.
+
+```python
```

Comment:
Colour formatting looks odd here, although the code is python, it's formatted as terminal output, maybe `text` based formatting is better here?